index.js

```diff
@@ -12,7 +12,7 @@ const normalizePackageData = require('normalize-package-data');
 
 // Prevent caching of this module so module.parent is always accurate
 delete require.cache[__filename];
-const parentDir = path.dirname(module.parent.filename);
+const parentDir = path.dirname(module.parent?.filename ?? '.');
```

Comment:
This syntax is not supported in the targeted Node.js version.

Reply:
Sorry, that was quick patch made in browser. Fixed and tested locally.